lib/samples/identify_raster_cell/identify_raster_cell.dart

```diff
@@ -0,0 +1,148 @@
+              controllerProvider: () => _mapViewController,
+              onMapViewReady: onMapViewReady,
+              onTap: onTap,
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            LoadingIndicator(visible: !_ready),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future<void> onMapViewReady() async {
+    // Define a new map with Wgs84 Spatial Reference.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISOceans);
+    map.initialViewpoint = Viewpoint.fromCenter(
+      ArcGISPoint(
+        x: -34.1,
+        y: 18.6,
+        spatialReference: SpatialReference.wgs84,
+        z: 9,
+      ),
+      scale: 10000,
+    );
+    // Add map to the map view.
+    _mapViewController.arcGISMap = map;
+    // Download the raster file.
+    await downloadSampleData(['b5f977c78ec74b3a8857ca86d1d9b318']);
+    // Get the application documents directory.
+    final appDir = await getApplicationDocumentsDirectory();
+    // Create a Raster from the local tif file.
+    final raster = Raster.withFileUri(
+      Uri.file(
+        '${appDir.absolute.path}/SA_EVI_8Day_03May20/SA_EVI_8Day_03May20.tif',
+      ),
+    );
+    // Create a Raster Layer.
+    _rasterLayer = RasterLayer.withRaster(raster);
+    // Load the Raster Layer.
+    await _rasterLayer.load();
+    // Add the Raster Layer to the map.
+    map.operationalLayers.add(_rasterLayer);
+    // Set the viewpoint.
+    if (_rasterLayer.fullExtent != null) {
+      await _mapViewController.setViewpointGeometry(_rasterLayer.fullExtent!);
+    }
+    _textGraphicsOverlay.graphics.add(_textGraphic);
+    _mapViewController.graphicsOverlays.add(_textGraphicsOverlay);
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  Future<void> onTap(Offset position) async {
+    // Get the result for where the user hovered on the raster layer.
+    final identifyResult = await _mapViewController.identifyLayer(
+      _rasterLayer,
+      screenPoint: position,
+      tolerance: 1,
+    );
+    // Create a StringBuffer to display information to the user.
+    final stringBuffer = StringBuffer();
+    // Get the identified raster cell.
+    if (identifyResult.geoElements.isNotEmpty) {
+      final cell = identifyResult.geoElements.first;
+      // Loop through the attributes (key/value pairs).
+      cell.attributes.forEach((key, value) {
+        stringBuffer.writeln('$key: $value');
+      });
+      // Get the x and y values of the cell.
+      if (cell.geometry != null) {
+        final x = cell.geometry!.extent.xMin;
+        final y = cell.geometry!.extent.yMin;
+        // Add the x & y coordinates where the user clicked raster cell to the string buffer.
+        stringBuffer.writeln();
+        stringBuffer.writeln('X: ${x.toStringAsFixed(4)}');
+        stringBuffer.write('Y: ${y.toStringAsFixed(4)}');
+
+        final textSymbol = TextSymbol(
```

Comment:
@jenmerritt - here's an interesting idea to use a TextSymbol instead of a Callout (until we implement Callout). We haven't used this before, but it makes sense here.

Reply:
Looks good. I don't think we need a rule as to what we suggest instead of Callout but it's another nice option as well as other UI elements 👍